apps/crm/src/modules/mailchimp/auth/mailchimp-auth-section/mailchimp-auth-section.tsx

```diff
@@ -0,0 +1,26 @@
+import { LoginWithMailchimpButton } from "../../../ui/login-with-mailchimp-button/login-with-mailchimp-button";
+import React, { useRef, useState } from "react";
+import { Box, Text } from "@saleor/macaw-ui/next";
+import { Section } from "../../../ui/section/section";
+import { AppBridgePersistence } from "../../../../lib/app-bridge-persistence";
+
+export const MailchimpAuthSection = () => {
+  const appBridgeContext = useRef(AppBridgePersistence.get());
```

Comment:
I suggest creating a separate hook in the `apps/crm/src/lib/app-bridge-persistence.ts` file, something like `usePersistentAppBridge`.

If you see this line of code first, you can get confused. If you abstract it away via hook, you can provide more context by its location & name.